### PR TITLE
Release1.9.0

### PR DESCRIPTION
**Pull request by xinlian12**
Kafka 1.9.0 release

### CODE REVIEW

**Review by kushagraThapar — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 ## Release History
 
+### 1.9.0 (2023-06-19)
+#### New Features
+* Updated `azure-cosmos` version to 4.46.0.
+
+#### Breaking Changes
+* This change will stop updating the lease container continuation token based on kafka offset when using the config `connect.cosmos.offset.useLatest`. [PR 516](https://github.com/microsoft/kafka-connect-cosmosdb/pull/516)
+
+When set to `false`, if the lease container exists, connector will now process the changes from the current continuationToken in `leaseContainer`, if the lease container does not exist, then it will start process the changes from beginning.
```
Comment (aayush3011) [inline CHANGELOG.md:10]: NIT: then it will start processing the changes from beginning.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+If you want to start from beginning, it is advised to delete the lease container or change the kafka worker name.
+
+when set to `true`, if the lease container exists, connector will now process the changes from the current continuation token in `leaseContainer`, if the lease container does exist, then it will start process the changes from now.
```
Comment (aayush3011) [inline CHANGELOG.md:13]: NIT: then it will start processing the changes from now.

**Review by aayush3011 — APPROVED** [review]
LGTM, just a couple of  change log comments.

File: CHANGELOG.md (hunk 2 above)
Comment (xinlian12) [reply CHANGELOG.md:13]: will resolve in a different PR

File: CHANGELOG.md (hunk 1 above)
Comment (xinlian12) [reply CHANGELOG.md:10]: will resolve in a different PR

**Pull request merged**